code/game/gamemodes/wizard/spells.dm

```diff
@@ -0,0 +1,739 @@
+/datum/spellbook_entry
+	var/name = "Entry Name"
+	var/is_ragin_restricted = FALSE // FALSE if this is buyable on ragin mages, TRUE if it's not.
+	var/spell_type = null
+	var/desc = ""
+	var/category = "Offensive"
+	var/cost = 2
+	var/refundable = TRUE
+	var/obj/effect/proc_holder/spell/S = null //Since spellbooks can be used by only one person anyway we can track the actual spell
+	var/buy_word = "Learn"
+	var/limit //used to prevent a spellbook_entry from being bought more than X times with one wizard spellbook
```

Comment:
Could you convert these comments into autodocced ones?

Reply:
40551a0